provider/provider_config/gcs_file_store_config_test.go

```diff
@@ -112,5 +114,12 @@ func TestGCSFileStoreDifferingFields(t *testing.T) {
 
 		})
 	}
+}
 
+func createMapCopy(originalMap map[string]interface{}) map[string]interface{} {
```

Comment:
Do we need this? Is Go pass by reference?

Reply:
yes, the test it is used would have failed otherwise.